src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
So basically whenever I update a string I have to also update all the translations right :smile: